doc/WALLET_DESKTOP.md

```diff
@@ -0,0 +1,104 @@
+# Wallet Desktop
+
+## Overview
+
+We will teach how to connect a ledger Nano S device with hathor wallet and how to operate the main features of the wallet with the ledger device.
+
+### Table of contents
+
+- [How to connect the wallet with your Nano S](#connect)
+- [View TX history](#view-tx-history)
+- [View TX Info](#view-tx-info)
+- [View Public Address](#view-public-address)
+- [Send HTR Tokens](#send-htr-tokens)
+
+## Connect
+
+First the device should be connected to the PC and then we should open the wallet (on the PC) and the hathor app (on Nano S).
+
+![Choose wallet screen](images/01-choose-type.png)
+
+Once you get to the screen above choose `Hardware wallet` and wait with the hathor app open, the wallet will find the device and check if the hathor app is open.
+
+![Second step](images/02-wait-find-device.png)
+
+When the wallet finds the Nano S the second step will begin, the wallet will ask permission to access the wallet public key.
+
+To confirm access, go to the `Approve` screen on the Nano S and press with both buttons.
+
+Now the wallet will load your addresses and transaction history, on the next screen you will be ready to send and receive HTR tokens.
+
+
+## View TX History
+
+The `Wallet` main screen has a list of previous transactions with basic info on the transaction and a link to view transaction info.
+
+![Wallet screen](images/03-wallet-screen.png)
+
+Information on transaction:
+
+- Date and time of the TX
+- Id of the transaction (with link to tx info page)
+- Type (Effect on wallet balance: send, receive or no effect)
+- Value (of the effect on wallet balance)
+
+This list paginates and will go on to the first transaction found on the wallet.
+
+## View TX Info
+
+![Tx info screen](images/06-tx-info-screen.png)
+
+By clicking on the TX id on the `View TX History` a page with all information on the transaction opens.
+
+Here you can check any information pertaining to the transaction.
+
+## View Public Address
+
+To receive tokens the sender must input an address to send the tokens.
+
+![Show address](images/04-show-full-address.png)
+
+Your wallet has many addresses, if you want to get one to receive tokens you need to:
+
+- On the `Wallet` main screen upper right corner there is a box with the current address
+- Click on "Show full address" to show the complete address, check your ledger device
+- On the ledger device a prompt to verify the address will appear, you should verify that the address on the screen matches the address on the device.
+- After checking the address you must click with both buttons on the `Verify` screen (on the ledger device)
+
+Any errors on the address will make you loose your tokens, only trust addresses that match the ledger generated address.
```

Comment:
"lose", with just 1 "o".

And split sentences.

Reply:
Done